### ISSUE TITLE
EditorConfig UI doesn't work correctly for VB-only style options

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#52696]
See https://github.com/dotnet/roslyn/pull/52695

**Comment by Youssef1313** [comment]
@jmarolf

**Comment by jmarolf** [comment]
@Youssef1313 there exist no VB-only style options today correct? Or was one merged and I missed it?

**Comment by CyrusNajmabadi** [comment]
I think we have one.  We have a style option if you like `Dim x as List(of integer) = new List(of integer)` versus `dim x as new List(of integer)`.

**Comment by Youssef1313** [comment]
> I think we have one. We have a style option if you like `Dim x as List(of integer) = new List(of integer)` versus `dim x as new List(of integer)`.

Yep, there are also few more I think (e.g, using IsNot expressions IIRC)

**Comment by paul1956** [comment]
Me vs. this qualifier would be another (with multiple sub options)
visual_basic_preferred_modifier_order

**Comment by CyrusNajmabadi** [comment]
Not planning on more investments here.
